dev-java/jna/jna-5.18.1.ebuild

```diff
@@ -121,6 +121,7 @@ src_test() {
 		-Djna.nosys=true
 		-Djna.boot.library.path=build/native
 		-Djna.library.path=build/native
+		--enable-native-access=ALL-UNNAMED
```

Comment:
What does this do, and why are we doing it now? The commit message doesn't explain it.

Reply:
It silences the warning:
```
.WARNING: A restricted method in java.lang.System has been called
WARNING: java.lang.System::load has been called by com.sun.jna.Native in an unnamed module (file:/var/tmp/portage/dev-java/jna-5.18.1/work/jna-5.18.1/jna.jar)
WARNING: Use --enable-native-access=ALL-UNNAMED to avoid a warning for callers in this module
WARNING: Restricted methods will be blocked in a future release unless native access is enabled
```

Reply:
I wonder if we will need this for some other JNI packages too in future.

Reply:
ACK.